util/chplenv/chpl_unwind.py

```diff
@@ -22,8 +22,8 @@ def get():
             return 'system'
     if osx:
         if val == 'libunwind':
-            raise ValueError("Using CHPL_UNWIND=libunwind is not supported"+
-                            " on Mac OS X. Use CHPL_UNWIND=system instead.")
+            error("Using CHPL_UNWIND=libunwind is not supported"+
```

Comment:
Oh.. I guess we were using a `+` here too. Can you get rid of this too (or am I super wrong about how often we use `+` just to join strings together